internal/dag/gatewayapi_processor.go

```diff
@@ -421,7 +429,12 @@ func (p *GatewayAPIProcessor) computeListener(
 	listener gatewayapi_v1beta1.Listener,
 	gwAccessor *status.GatewayStatusUpdate,
 	validateListenersResult gatewayapi.ValidateListenersResult,
-) (bool, *listenerInfo) {
+) (info *listenerInfo) {
```

Comment:
We prefer not to use named return params, let's switch this to non-named and just return the local variable where needed.